actions/create_embed_message.js

```diff
@@ -259,44 +252,20 @@ module.exports = {
 						embed.setTimestamp(new Date(`${text}`));
 					} else {
 						embed.setTimestamp(new Date());
```

Comment:
```suggestion
						embed.setTimestamp();
```

Reply:
Honestly I disagree with this suggestion.